edit/linter-stylelint.js

```diff
@@ -0,0 +1,229 @@
+      'declaration-colon-newline-after': 'always-multi-line',
+      'declaration-colon-space-after': 'always-single-line',
+      'declaration-colon-space-before': 'never',
+      'declaration-empty-line-before': [
+        'always',
+        {
+          'except': [
+            'after-declaration',
+            'first-nested'
+          ],
+          'ignore': [
+            'after-comment',
+            'inside-single-line-block'
+          ]
+        }
+      ],
+      'function-comma-newline-after': 'always-multi-line',
+      'function-comma-space-after': 'always-single-line',
+      'function-comma-space-before': 'never',
+      'function-max-empty-lines': 0,
+      'function-name-case': 'lower',
+      'function-parentheses-newline-inside': 'always-multi-line',
+      'function-parentheses-space-inside': 'never-single-line',
+      'function-whitespace-after': 'always',
+      'indentation': 2,
+      'length-zero-no-unit': true,
+      'max-empty-lines': 1,
+      'media-feature-colon-space-after': 'always',
+      'media-feature-colon-space-before': 'never',
+      'media-feature-name-case': 'lower',
+      'media-feature-parentheses-space-inside': 'never',
+      'media-feature-range-operator-space-after': 'always',
+      'media-feature-range-operator-space-before': 'always',
+      'media-query-list-comma-newline-after': 'always-multi-line',
+      'media-query-list-comma-space-after': 'always-single-line',
+      'media-query-list-comma-space-before': 'never',
+      'no-eol-whitespace': true,
+      'no-missing-end-of-source-newline': true,
+      'number-leading-zero': 'always',
+      'number-no-trailing-zeros': true,
+      'property-case': 'lower',
+      'rule-empty-line-before': [
+        'always-multi-line',
+        {
+          'except': [
+            'first-nested'
+          ],
+          'ignore': [
+            'after-comment'
+          ]
+        }
+      ],
+      'selector-attribute-brackets-space-inside': 'never',
+      'selector-attribute-operator-space-after': 'never',
+      'selector-attribute-operator-space-before': 'never',
+      'selector-combinator-space-after': 'always',
+      'selector-combinator-space-before': 'always',
+      'selector-descendant-combinator-no-non-space': true,
+      'selector-list-comma-newline-after': 'always',
+      'selector-list-comma-space-before': 'never',
+      'selector-max-empty-lines': 0,
+      'selector-pseudo-class-case': 'lower',
+      'selector-pseudo-class-parentheses-space-inside': 'never',
+      'selector-pseudo-element-case': 'lower',
+      'selector-pseudo-element-colon-notation': 'double',
+      'selector-type-case': 'lower',
+      'unit-case': 'lower',
+      'value-list-comma-newline-after': 'always-multi-line',
+      'value-list-comma-space-after': 'always-single-line',
+      'value-list-comma-space-before': 'never',
+      'value-list-max-empty-lines': 0
+      */
+    }
+  };
+  let config;
+
+  const prepareConfig = cacheFirstCall(() => {
```

Comment:
One file for the defaults could work, of course, but we had two previously and it was okay I think. Can't you make it work with two files for the defaults and one (like we had previously) or two for the prepare code?

Reply:
I prefer to create separated modules for different linters. In case we are going to add more linters, we can create `linter-{linterName}.js` to write new code. `linter-defaults.js` + `linter-engines.js` would work but it is not that clean IMHO.

Reply:
well, we can refactor it when (and if) we have three linters.